xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -84,6 +91,16 @@
   // responses.
   private final Map<String, String> routeConfigNamesToClusterNames = new HashMap<>();
 
+  // Cached data for EDS responses, keyed by cluster names.
+  // CDS responses indicate absence of clusters and RDS responses indicate presence of clusters.
```

Comment:
Is this right that "CDS responses indicate absence of clusters" ? CDS contains info about a single cluster but why does it indicate absence of clusters? Also should RDS be EDS ? Some clarification needed.

Reply:
Each CDS response contains a list of resources, each resource represents a cluster. Yes, RDS should be EDS, this is a typo. Thanks.